### PR TITLE
manga-ocr: 0.1.12 -> 0.1.13

### PR DESCRIPTION
**Pull request by Laurent2916**
https://github.com/kha-white/manga-ocr

- Moved the package to `pkgs/by-name/`
- Updated to 0.1.13

Followup of #349961

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by DerDennisOP — APPROVED** [review]
lgtm

File: pkgs/by-name/ma/manga-ocr/package.nix [hunk 1]
```diff
@@ -8,15 +8,15 @@ with python3Packages;
 
 buildPythonPackage rec {
   pname = "manga-ocr";
-  version = "0.1.12";
+  version = "0.1.13";
   disabled = pythonOlder "3.7";
   format = "pyproject";
```
Comment (DerDennisOP) [inline pkgs/by-name/ma/manga-ocr/package.nix:13]: format is depricated. please change it as suggested
```suggestion
  pyproject = true;
```

File: pkgs/by-name/ma/manga-ocr/package.nix [hunk 2]
```diff
@@ -8,15 +8,15 @@ with python3Packages;
```
Comment (DerDennisOP) [inline pkgs/by-name/ma/manga-ocr/package.nix:8]: with python3Packages is also deprecated

**Comment by Laurent2916** [comment]
Hi @DerDennisOP, thanks for the review!
I should probably keep myself better informed on deprecations and new best practices.

**Comment by DerDennisOP** [comment]
> Hi @DerDennisOP, thanks for the review! I should probably keep myself better informed on deprecations and new best practices.

now formatting is of, I dont like the new formatter but we gotta stick to it. please use the nix-fmt-rfc-style formatter to format your file

**Comment by teto** [comment]
and the reference from python-modules should be removed

**Review by DerDennisOP — APPROVED** [review]
lgtm

**Review by FliegendeWurst — CHANGES_REQUESTED** [review]
Python modules should not be moved to pkgs/by-name.

**Comment by teto** [comment]
> Python modules should not be moved to pkgs/by-name.

why is that ? It's a python application here, if it makes any difference

**Comment by FliegendeWurst** [comment]
> why is that ? It's a python application here, if it makes any difference

It is both a library and an application (judging by the readme). In that case the module should stay in python-modules, but you can add a wrapper in pkgs/by-name.

**Comment by Laurent2916** [comment]
Updated in https://github.com/NixOS/nixpkgs/pull/360641

**Pull request closed without merging**